config/initializers/e7a_1418_iframe.rb.example

```diff
@@ -0,0 +1,4 @@
+# # This file allows you to specify the location of the e7a_1418_iframe.js file in order to load a specific version.
```

Comment:
config/initializers/e7_1418_iframe.rb needs to be accounted for in the deployment scripts deploy/before_bundle.rb and deploy/before_symlink.rb to ensure that it is propagated across deployments and from app master to slaves in the EY environments.